CONTRIBUTING.md

```diff
@@ -1,3 +1,20 @@
+## Steps for Contributing
+1. Create a new branch off of up-to-date master
+2. Make your changes.  If you are adding an endpoint or resource to the API,
+follow "Adding an API Resource"
+3. Make sure your code follows the "Code Standards" section of this document
+4. Use "git rebase -i origin/master" to squash your commits into one or a few. Follow commit message guidelines.
```

Comment:
If this document is targeting external contributors, I would be tempted to remove this step. IMO it's easier to get some contribution going, then ask for this kind of stuff at the end if necessary.